### PR TITLE
Change inverse computation to lazy-computation speeding up initialization

### PR DESCRIPTION
**Pull request by erykoff**
The inverse isn't actually needed all the time, and this speeds up initialization by a factor of approximately x133.

### CODE REVIEW

**Comment by esheldon** [comment]
Thanks Eli.

Is this new keyword needed? Can we instead just set the inverse to None in the constructor, and then compute it on the fly when it is needed by checking if it is None?

**Comment by erykoff** [comment]
If we want it to always be lazy (not a bad idea) then that would make more sense.

**Comment by esheldon** [comment]
What do you think?  Can we simplify and remove the keyword?

**Comment by erykoff** [comment]
I was thinking about this this weekend.  And I wonder if there are any times that somebody would want to make sure that they can force the inverse calculation on initialization instead on demand.  I'm guessing that this is a vanishingly small use case.  Should I slim this down to do it completely on demand?

**Comment by esheldon** [comment]
I vote yes, slim it down.

**Comment by esheldon** [comment]
thanks Eli

File: esutil/wcsutil.py [hunk 1]
```diff
@@ -1060,11 +1066,10 @@ def ExtractDistortionModel(self):
...
 
-                # If inverse not there, calculate it
-                if cap == 0 or cbp == 0:
-                    self.InvertDistortion()
-                    self.distort['ap_order'] = self.distort['a_order']+1
-                    self.distort['bp_order'] = self.distort['b_order']+1
+                # If inverse can't be computed, treat it as if it
+                # already were computed
+                if cap == 0 and cbp == 0:
+                    self._inverse_computed = True
```
Comment (beckermr) [inline esutil/wcsutil.py:1072]: This is the bug. cc @esheldon @erykoff 

related to https://github.com/esheldon/esutil/issues/68

**Pull request merged**